index.html

```diff
@@ -90,7 +116,20 @@ <h2 class="weekday"> Welcome! </h2>
         </div>
 
     </div>
+
+    <!-- Footer -->
+    <footer class="text-center py-3 mt-auto">
+        <div class="container">
+            <hr class="vertical-line">
+            <p>&copy; 2023 bszet.me<br> All rights reserved.<br>
```

Comment:
Btw, I suggest removing that line, there are literally no rights or whatsoever we could reserve for ourselves.

Reply:
Nah, I don't think that's a problem. And even if "all right reserved" wouldn't do anything to that, would need to be something like "all information are supplied without liability and all obligation".

Reply:
then lets take this one